riva/client/argparse_utils.py

```diff
@@ -78,6 +78,12 @@ def add_asr_config_argparse_parameters(
         type=float,
         help="Threshold value for detecting the end of speech utterance",
     )
+    parser.add_argument(
+        "--stop-eou-threshold",
+        default=-1.0,
+        type=float,
+        help="Threshold value for likelihood of blanks before detecting end of utterance",
+    )
```

Comment:
can you update help for `x-history` parameters to indicate the unit ? currently it doesn't mention. I assume it is in seconds or miliseconds ?

Reply:
Updated!